packages/editor/src/components/table-of-contents/index.js

```diff
@@ -19,7 +20,7 @@ function TableOfContents( { hasBlocks, hasOutlineItemsDisabled } ) {
 			renderToggle={ ( { isOpen, onToggle } ) => (
 				<Button
 					onClick={ hasBlocks ? onToggle : undefined }
-					icon="info-outline"
```

Comment:
Let's go with just "info" on this one. It's not a safe bet that we'll ever have a "not outline" version of this icon, and if it ever comes to pass we have two, it won't be the end of the world if this icon goes solid.